pepper-apis/src/main/java/org/broadinstitute/ddp/db/dao/JdbiUserStudyEnrollment.java

```diff
@@ -196,9 +196,10 @@ int deleteByUserGuidStudyGuid(@Bind("userGuid") String userGuid,
                     + " JOIN umbrella_study us ON us.umbrella_study_id = usen.study_id "
                     + " AND usen.user_id = :userId"
                     + " AND usen.study_id = :studyId"
+                    + " ORDER BY usen.valid_from desc"
```

Comment:
Just curious: Looks like we are streaming through all status rows to fond if enrolled before or not.
Are we gaining much with this sort !

Reply:
We are streaming, but this sort here means we don't have to do a sort on the stream to get the latest enrollment status.